pandas-stubs/core/generic.pyi

```diff
@@ -300,7 +302,7 @@ class NDFrame(indexing.IndexingMixin):
         method: _str | None = ...,
         copy: _bool = ...,
         limit=...,
-        tolerance=...,
+        tolerance: Scalar | AnyArrayLike | Sequence[Scalar] = ...,
```

Comment:
Since `reindex_like` is defined in both `series.pyi` and `frame.pyi`, we can remove it here.  Can you do that in this PR?